core/google/cloud/future/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017, Google Inc.
```

Comment:
Do you intend to make the `google.cloud.operation` module a backward-compatibilitiy-only shim?